extensions/wikia/ArticleVideo/scripts/ooyalaVideo.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * If we want to continue using ooyala player for "premium videos"
+ * we should merge this with ooyala handler (Ooyala.js from VideoHandlers dir)
+ */
+define('ooyalaVideo', ['jquery', 'wikia.window', 'wikia.loader'], function ($, window, loader) {
+
+	function OoyalaVideo(elementId, ooyalaJsFile, ooyalaVideoId, onCreate) {
+		console.log('ooyala constructor');
+		var self = this,
+			ooyalaPlayerOptions = {
+				autoplay: false,
+				width: '100%',
+				height: '100%',
+				onCreate: onCreate
+			};
+
+		console.log('ooyala js loaded');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)